articles/libraries/lock/v10/index.md

```diff
@@ -82,15 +81,15 @@ lock.on("authenticated", function(authResult) {
 });
 ```
 
-### Showing Lock
+### 2) Showing Lock
 
```

Comment:
A sentence at the beginning always helps. Add something short but descriptive, for example: "This script displays the lock widget once the user clicks on the login button". This goes for all examples.